cyclus_input_gen/from_pris.py

```diff
@@ -306,26 +304,26 @@ def reactor_render(reactor_data, output_file, is_cyborg=False):
                 'assemblies_per_core': 764 / 1000.0,
                 'assemblies_per_batch': 764 / 3000.0}
     phwr_spec = {'template': candu_template,
-                 'kg_per_assembly': 8000 / 473,
-                 'assemblies_per_core': 473 / 500.0,
-                 'assemblies_per_batch': 60}
+                 'kg_per_assembly': 8000 / 473 / 60.0,
+                 'assemblies_per_core': 473 / 500.0 / 60.0,
+                 'assemblies_per_batch': 1}
     candu_spec = {'template': candu_template,
-                  'kg_per_assembly': 8000 / 473,
-                  'assemblies_per_core': 473 / 500.0,
-                  'assemblies_per_batch': 60}
+                  'kg_per_assembly': 8000 / 473 / 60.0,
+                  'assemblies_per_core': 473 / 500.0 / 60.0,
+                  'assemblies_per_batch': 1}
     pwr_spec = {'template': pwr_template,
                 'kg_per_assembly': 446.0,
                 'assemblies_per_core': 193 / 1000.0,
                 'assemblies_per_batch': 193 / 3000.0}
     epr_spec = {'template': pwr_template,
                 'kg_per_assembly': 467.0,
-                'assemblies_per_core': 216,
-                'assemblies_per_batch': 72}
+                'assemblies_per_core': 216 / 1670.0,
+                'assemblies_per_batch': 72 / 1670.0}
 
     reactor_specs = {'AP1000': ap1000_spec,
-                     'PHWR': phwr_spec,
+                     # 'PHWR': phwr_spec,
```

Comment:
why comment rather than delete?

Reply:
oh, this is supposed to be uncommented, I will fix the error.